scripts/functions.js

```diff
@@ -13,9 +13,9 @@ var urlParams = {};
 window.location.href.replace(
 	/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {urlParams[key] = value;}
 );
-var showHouseDistricts = true;
+var showHouseDistricts = false;
 var showSenateDistricts = false;
-var showSchoolDistricts = false;
+var showSchoolDistricts = true;
```

Comment:
It'll probably make sense to remove all of the House / Senate districts logic from this at the end, but for now I'm keeping it in and just disabled in case CBF decide they do want to see those after all.

Reply:
Yes I will bet you a pint of Seattle's finest microbrew that CBF will add that stuff (or similar) back in before we're through